src/app/categories/categories.repository.ts

```
@@ -0,0 +1,45 @@
+        );
+    }
+
+    // Delete a category post by its ID
+    static async deleteById(id: string): Promise<ICategory | null> {
+        return await Category.findByIdAndDelete(id);
+    }
+
+    static async findByName(name: string): Promise<ICategory | null> {
+        return await Category.findOne({ name });
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)